lightning/src/ln/functional_tests.rs

```diff
@@ -9376,11 +9189,7 @@ fn test_keysend_payments_to_private_node() {
 	nodes[1].node.peer_connected(&payer_pubkey, &msgs::Init { features: InitFeatures::known() });
 
 	let _chan = create_chan_between_nodes(&nodes[0], &nodes[1], InitFeatures::known(), InitFeatures::known());
-	let network_graph = &nodes[0].net_graph_msg_handler.network_graph;
-	let first_hops = nodes[0].node.list_usable_channels();
-	let route = get_keysend_route(&payer_pubkey, &network_graph, &payee_pubkey,
```

Comment:
This seems to be the only place we call `get_keysend_route` in tests through a private channel, so I think best to keep it.

Reply:
Reverted and updated the other keysend test to use `get_keysend_route`.